app/src/main/java/calculator/domain/BaseCalculation.java

```diff
@@ -0,0 +1,23 @@
+package calculator.domain;
+
+public class BaseCalculation implements Calculation {
+    @Override
+    public int addition(int a, int b) {
+        return a + b;
+    }
+
+    @Override
+    public int subtraction(int a, int b) {
+        return a - b;
+    }
+
+    @Override
+    public int multiplication(int a, int b) {
+        return a * b;
+    }
+
+    @Override
+    public int division(int a, int b) {
+        return a / b;
```

Comment:
b=0인경우 ArithmeticException이 발생하는 것을 BaseCalculatorService에서 잡아주는군요!

해당 부분에 더 명시적으로 어느경우에 발생하는지 조건과 함께 직접 명시해 주는 것이 좋다고 생각합니다!

```suggestion
        if(...){
            throw new ...
        }
        return a / b;
```

Reply:
명시적으로요! 확인했습니다~